content/en/docs/event-sources/drivers.md

```diff
@@ -7,19 +7,55 @@ Falco depends on a driver that taps into the stream of system calls on a machine
 
 The kernel module called `falco` is the default driver. Alternatively, an eBPF probe can be used. 
 
-# Using the kernel module
+The Falco project has three different kind of drivers.
 
+- Kernel module
+- eBPF probe
+- Userspace instrumentation
+
+## Kernel module
 
 By default, the kernel module will be installed when installing the Falco [debian/rpm](/docs/installation) package, when running the `falco-driver-loader` script shipped within the [binary pagacke](/docs/installation#linux-binary), or when running the `falcosecurity/falco-driver-loader` docker image (that just wraps the aforementioned script). 
 
 To install the kernel module, please refer to the [installation](/docs/installation/#install-driver) page.
 
 Falco tries to use the kernel module driver by default.
 
-# Using the eBPF probe
+## eBPF probe
 
 The eBPF probe is an alternative to the one described above. Note that eBPF is a feature supported only by recent kernels.
 
 To install the eBPF probe, please refer to the [installation](/docs/installation/#install-driver) page.
 
 To enable the eBPF support in Falco set the `FALCO_BPF_PROBE` environment variable to an empty value (ie. `FALCO_BPF_PROBE=""`) or otherwise explicitly set it to the path where the eBPF probe resides.
+
+## Userspace instrumentation
+
+Differently from the other drivers, as the name suggests, userspace instrumentations happens 100% in userspace.
```

Comment:
Should we mention details about /proc scanning?

Just a thought.

Reply:
I don't think so.